platform/rpc/nats/bus.go

```diff
@@ -332,7 +340,6 @@ func (n *NATS) Subscribe(instID flux.InstanceID, remote platform.Platform, done
 			// consequence of asynchronous request handling. The error will get
 			// selected and handled soon enough.
 			case err := <-errc:
-				close(errc)
```

Comment:
This will get garbage collected when it goes out of scope, and nothing is synchronised on it.